src/lib/homestore/heap_chunk_selector.h

```diff
@@ -0,0 +1,46 @@
+#pragma once
+
+#include <homestore/chunk_selector.h>
+#include <homestore/vchunk.h>
+#include <homestore/homestore_decl.hpp>
+
+#include <queue>
+#include <vector>
+#include <mutex>
+#include <utility>
+#include <memory>
+
+#include <folly/concurrency/ConcurrentHashMap.h>
+
+namespace homeobject {
+
+using csharedChunk = homestore::cshared<homestore::Chunk>;  
```

Comment:
I don't understand this `cshared` naming convention? I see it's being used in HomeStore but why `c`? Seems names like `chunk_ptr` would be better? Then we don't have to change the code everywhere if we switch from shared to unique or raw...etc

Reply:
actually, i have the similar idea. but , this is from homestore, maybe we can create a pr in homestore to make this change